sandbox/api/FHIR/R4/index.js

```diff
@@ -0,0 +1,31 @@
+import { Router } from 'express';
+import response0 from './responses/empty-response.fhir';
+import response1 from './responses/1_exempt-vaccine.fhir';
+import response2 from './responses/2_exempt-vaccine-and-testing.fhir';
+import response3 from './responses/3_declined-vaccine.fhir';
+import response4 from './responses/4_declined-vaccine-and-testing.fhir';
+const responses = [
+    response0,
+    response1,
+    response2,
+    response3,
+    response4
+];
+
+export default function(dependencies, config) {
+    const router = new Router();
+    
+    router.get('/CovidMedicalExemption', (req, res, next) => {
+        try {
+            const [type, id] = req.query['patient.identifier'].split('|');
```

Comment:
Makes sense debating if we need to lock down the nhs numbers or not in a dict mapping?